.github/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+Your PR title must follow ![conventional commits](https://github.com/Uniswap/interface/pull/3832/files), and should start with one of the following ![types](https://github.com/Uniswap/interface/pull/3832/files):
+
+- build: Changes that affect the build system or external dependencies (example scopes: gulp, broccoli, npm)
```

Comment:
Replace the scopes (and below) with scopes relevant to our own processes

Reply:
LOL i pasted the wrong thing... my bad